src/components/Signup.js

```diff
@@ -0,0 +1,98 @@
+import React, { Component, PropTypes } from 'react'
+import rokka from '../rokka'
+import FramelessLayout from './layouts/FramelessLayout'
+import { login } from '../state'
+import {setAlert} from '../state/index'
+
+class Signup extends Component {
+
+  constructor () {
+    super()
+
+    this.state = {
+      email: '',
+      organization: '',
+      showTransition: false
+    }
+
+    this.onSubmit = this.onSubmit.bind(this)
+  }
+
+  onSubmit (e) {
+    e.preventDefault()
+    rokka.users.create(this.state.email, this.state.organization).then(response => {
+      login(this.state.organization, response.body.api_key, (done) => {
+        this.props.router.push('/')
```

Comment:
can you add here a `success` alert with a message similar to: "Successfully signed up. You will receive an email with the credentials"?